packages/sdks/snippets/react/src/routes/query-cheatsheet/queries/gte.tsx

```diff
@@ -0,0 +1,11 @@
+import { fetchOneEntry } from '@builder.io/sdk-react';
+
+export const getProduct = async (options: {
+  apiKey: string;
+  [key: string]: unknown;
+}) =>
+  fetchOneEntry({
+    model: 'product',
+    query: { 'data.price': { $gte: 700 } },
+    ...options,
+  });
```

Comment:
you should rely on `GetContentOptions` instead of `unknown` to let types flow from the SDK

```suggestion
  import { fetchOneEntry, GetContentOptions } from '@builder.io/sdk-react';
  
  const getProduct = async (options: GetContentOptions) =>
    fetchOneEntry({
      query: { 'data.price': { $gte: 700 } },
      ...options,
    });
```

same applies to gen1 SDK (remove `unknown`). You'll have to rework the logic a bit to provide `model` from the component for your tests

Reply:
Also, the `async` are redundant here and can be removed. They are only useful if you use `await` in the body.

Reply:
Thanks @samijaber. Those changes have been made -- please let me know if you have any other thoughts!